packages/blade/src/utils/getFocusRingStyles/getFocusRingStyles.web.ts

```diff
@@ -1,15 +1,25 @@
-import type { CSSProperties } from 'styled-components';
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
 import type { Theme } from '../../components/BladeProvider';
 import { castWebType, makeMotionTime } from '~utils';
 
-function getFocusRingStyles(theme: Theme): CSSProperties {
+type GetFocusRingProps = {
+  theme: Theme;
+  negativeOffset?: boolean;
+};
+
+/**
+ * @param props.theme Blade Theme Object
+ * @param props.negativeOffset if set the outline offset will be set to -4px, this is useful
```

Comment:
How were we doing this earlier? I think I was using some CSS workaround only for this. Was it offset only?

Reply:
We used `inset` box shadows.